TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -283,5 +283,22 @@ private DatetimeConstant CreateDatetimeConstant(CodeElementsParser.Datetime_cons
 
             return null;
         }
+        public SavepointStatement CreateSavepointStatement(CodeElementsParser.SavepointStatementContext context)
+        {
+            if (context == null) return null;
+            SymbolReference savepointName = null;
+            SyntaxProperty<bool> retainLocks = null;
+            if (context.savepoint_name != null)
+            {
+                savepointName=new SymbolReference(new AlphanumericValue(context.savepoint_name as Token), SymbolType.SqlIdentifier);
```

Comment:
spacing

Reply:
use direct cast: `(Token) context.savepoint_name`, it's slightly faster and we do not need any checks here.